lib/collections.js

```diff
@@ -41,7 +55,9 @@ var uniqueWords = [];
 
 // The longest and shortest word in the paragraph
 var longAndShort = {};
-
+for (var i = 0, i < words.length; ++1) {
+  longAndShort.length[uniquewords]
+}
```

Comment:
This code is not syntactically correct JavaScript, and it's not clear what you were trying to do.